docs/source/examples/combine_techniques.md

```diff
@@ -0,0 +1,107 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Composing techniques: Readout Error Mitigation and Zero Noise Extrapolation
+
+Noise in quantum computers can arise from a variety of sources, and sometimes applying multiple error mitigation techniques can be more beneficial than applying a single technique alone. Here we apply a combination of readout error mitigation (REM) and zero noise extrapolation (ZNE) to a randomized benchmarking (RB) task. In REM, the inverse transition / confusion matrices are generated and applied to the noisy measurement results. In ZNE, the expectation value of the observable of interest is computed at different noise levels, and subsequently the ideal expectation value is inferred by extrapolating the measured results to the zero-noise limit. More information on the [REM](../guide/rem.md) and [ZNE](../guide/zne.md) techniques can be found in the corresponding sections of the user guide.
+
++++
+
+## Setup
+
+We begin by importing the relevant modules and libraries required for the rest of this tutorial.
+
+```{code-cell} ipython3
+import cirq
+import numpy as np
+from mitiq.benchmarks import generate_rb_circuits
+from mitiq import MeasurementResult, Observable, PauliString, raw
+```
+
+## Task
+
+We will demonstrate using REM + ZNE on RB circuits, which are generated using Mitiq's built-in benchmarking function, `generate_rb_circuits`. More information on the RB protocol is available [here](https://learn.qiskit.org/course/quantum-hardware/randomized-benchmarking). In this example we use a two-qubit RB circuit with a Clifford depth (number of Clifford groups) of 5.
```

Comment:
Link to API-doc on `generate_rb_circuits` could be helpful: https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.benchmarks.randomized_benchmarking.generate_rb_circuits

Reply:
Agreed. Hopefully the way I'm referencing it now works on RTD.